pkg/controllers/job/job_controller_actions.go

```diff
@@ -845,6 +839,29 @@ func (cc *jobcontroller) initJobStatus(job *batch.Job) (*batch.Job, error) {
 	return newJob, nil
 }
 
+func (cc *jobcontroller) recordPodGroupEvent(job *batch.Job, podGroup *scheduling.PodGroup) {
+	var latestCondition scheduling.PodGroupConditionType
+	var latestTransitionTime *metav1.Time
+	var latestConditionMsg string
+
+	// Get the latest condition by timestamp
+	for _, condition := range podGroup.Status.Conditions {
```

Comment:
A little doubt here, the unschedulable condition will append to unschedulable condition？What if just override the unschedulable condition because pg is already schedulable and no need to concern the unschedulable condition.

Reply:
This is just to solve the problem that the existing podgroup will repeatedly log warning events when the controller restarts. The condition update issue needs to be fixed separately.

Reply:
Maybe we can add a ut here.

Reply:
UT testcase has been added